client/src/App.js

```diff
@@ -1,8 +1,33 @@
 import React from 'react';
+import { BrowserRouter as Router, Link, Switch } from 'react-router-dom';
 import './App.css';
+import Pages from './pages';
 
 function App() {
-  return <div className="App" />;
+  return (
+    <div className="App">
+      <Router>
+        <div>
+          <nav>
+            <ul>
+              <li>
+                <Link to="/">index</Link>
+              </li>
+              <li>
+                <Link to="/dashboard">dashboard</Link>
+              </li>
+              <li>
+                <Link to="/profile">profile</Link>
+              </li>
+            </ul>
+          </nav>
+          <Switch>
+            <Pages />
```

Comment:
Not what i meant, but a really nice effort,
I want the pages routes to be in this page, for easier code navigation, 
```jsx
     <Route path="/dashboard">
        <Dashboard />
      </Route>
      <Route path="/profile">
        <Profile />
      </Route>
      <Route path="/">
        <Home />
      </Route>
```
i want this code here, and the pages index to be:
export {default as <name> } from './file'